processing/src/main/java/org/apache/druid/query/QueryContexts.java

```diff
@@ -35,6 +35,7 @@
 public class QueryContexts
 {
   public static final String PRIORITY_KEY = "priority";
+  public static final String LANE_KEY = "lane";
```

Comment:
No, I hadn't documented since I hadn't decided the behavior of whether or not a lane specified in the query context by the user should override the laning strategy, or if it should be laning strategy specific, see other comment about this